### DISCUSSION TITLE
Index "Date fields use deprecated camel case formats"

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#1031] [Q&A] [answered]
Trying to resolve any critical issues in "Upgrade Assistant" on Kibana:

```
Date fields use deprecated camel case formats

Index: elastalert_status_status

Convert [@timestamp] format [dateOptionalTime] which contains deprecated camel case to snake case. [dateOptionalTime] to [date_optional_time]
```

I already changed to "date_optional_time" in all JSON files located in `es_mapping/7` but when i run `elastalert-create-index` the index is still using the older value and the errors still persist

**Comment by jertel** [comment, marked as answer]
You will need to delete the indexes first, since they do not get recreated if they already exist. Another option might be to upgrade to Elastic 8, since the mappings are already corrected for that version.

**Reply by ngms17** [reply]
Yes, i deleted the indexes before running the command

**Reply by jertel** [reply]
Are you running the Docker container version of ElastAlert 2?

**Reply by ngms17** [reply]
No, not running the docker version

**Reply by jertel** [reply]
Did you use pip to install ElastAlert 2 on your host machine? If so, perhaps you are editing the GitHub project mapping file, and not the installed mapping file under the `/usr/...` dir.

**Reply by ngms17** [reply]
Yup! That seems to resolve this issue.

Thanks!
